cmake/configs/qurt_eagle_release.cmake

```diff
@@ -70,7 +70,7 @@ set(config_module_list
 	modules/uORB
 	modules/commander
 	modules/controllib
-
+	
```

Comment:
git checkout <hash-of-commit-with-old-version> cmake/configs/qurt_eagle_release.cmake